src/styles/block/nav.scss

```
@@ -0,0 +1,42 @@
+.nav {
+  &__list {
+    list-style: none;
+    display: flex;
+    align-items: center;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    text-decoration: none;
+    color: #060b35;
+    font-weight: 500;
+    font-size: 1.2em;
+    line-height: 14px;
+    text-transform: uppercase;
+    height: 60px;
+    transition: color 0.3s ease-in-out;
```

Create variable for transition duration
```suggestion
    transition: color $transitionDuration ease-in-out;
```